strr-web/components/bcros/form-section/review/Form.vue

```diff
@@ -75,46 +75,67 @@
           <p class="font-bold mb-6 m:mx-2">
-            {{ tReview('review') }}
+            {{ tReview('confirmation') }}
           </p>
           <div class="bg-white p-8 mb-6">
-            <UCheckbox
-              v-model="formState.principal.agreeToSubmit"
-              :label="tReview('confirm')"
-              :ui="{ label: isComplete && !formState.principal.agreeToSubmit ? 'text-bcGovColor-error' : '' }"
-            />
-          </div>
-          <div
-            v-if="formState.isPropertyManagerRole"
-            class="bg-white p-8 mb-6"
-            data-test-id="host-auth-checkbox"
-          >
-            <UCheckbox
-              v-model="formState.hasHostAuthorization"
-              :label="tReview('confirmHostAuthorization')"
-              :ui="{ label: isComplete && !formState.hasHostAuthorization ? 'text-bcGovColor-error' : '' }"
-            />
+            <div class="space-y-6">
+              <div data-test-id="terms-and-conditions-confirmation">
+                <p>
+                  <span class="font-bold">{{ tReview('confirmTermsAndConditionsLabel') }}</span>
+                  {{ tReview('confirmTermsAndConditionsFirst') }}
+                  <a
+                    class="text-bcGovColor-activeBlue"
+                    target="_blank"
+                    :href="`
+                      https://www2.gov.bc.ca/gov/content/housing-tenancy/\
+                        short-term-rentals/terms-and-conditions
+                    `"
+                    rel="noopener noreferrer"
+                  >
+                    {{ tReview('confirmTermsAndConditionsLink') }}
+                  </a>
+                  {{ tReview('confirmTermsAndConditionsSecond') }}
+                </p>
+                <hr class="mt-4 border-gray-500">
+              </div>
+              <div data-test-id="tax-auditing-confirmation">
+                <p>
+                  <span class="font-bold">{{ tReview('confirmTaxAuditingLabel') }}</span>
+                  {{ tReview('confirmTaxAuditing') }}
+                </p>
+                <hr class="mt-4 border-gray-500">
+              </div>
+              <div data-test-id="info-accuracy-confirmation">
+                <p>
+                  <span class="font-bold">{{ tReview('confirmInfoAccuracyLabel') }}</span>
+                  {{ tReview('confirmInfoAccuracy') }}
+                </p>
+                <hr class="mt-4 border-gray-500">
+              </div>
+              <div v-if="formState.isPropertyManagerRole" data-test-id="host-auth-confirmation">
+                <p>
+                  <span class="font-bold">{{ tReview('confirmHostAuthorizationLabel') }}</span>
+                  {{ tReview('confirmHostAuthorization') }}
+                </p>
+                <hr class="mt-4 border-gray-500">
+              </div>
+              <UCheckbox
+                v-model="confirmationCheckboxValue"
+                label="I confirm that I understand and agree to the above."
```

Comment:
It should come from the translation file.

Reply:
Fixed